client/dashboard/task-list/tasks/tax.js

```diff
@@ -369,16 +378,22 @@ class Tax extends Component {
 }
 
 export default compose(
-	withSelect( select => {
+	withSelect( ( select, props ) => {
 		const {
 			getActivePlugins,
 			getOptions,
 			getSettings,
 			getSettingsError,
 			isGetSettingsRequesting,
+			isGetJetpackConnectUrlRequesting,
 			isJetpackConnected,
 		} = select( 'wc-api' );
 
+		const jetpackQueryArgs = {
```

Comment:
This works well but it would be great if we could avoid duplicating logic outside the `Connect` component in case it ever changes.

What do you think about a `setRequesting` prop or `onRequesting` and `onReady` prop for the `Connect` component so that we can set `isPending` to `true|false` inside the tax component?